.changeset/lovely-walls-guess.md

```diff
@@ -0,0 +1,90 @@
+---
+'@backstage/create-app': patch
+---
+
+Added `SidebarSubmenu` for all Kinds making it easier to discover them in the Catalog.
+
+To add this to an existing Backstage instance you will need to make the following to your `packages/app/src/components/Root/Root.tsx`:
+
+1. First add the following imports:
+
+   ```ts
+   import ApiIcon from '@material-ui/icons/Extension';
+   import ComponentIcon from '@material-ui/icons/Memory';
+   import DomainIcon from '@material-ui/icons/Apartment';
+   import ResourceIcon from '@material-ui/icons/Work';
+   import SystemIcon from '@material-ui/icons/Category';
+   import UserIcon from '@material-ui/icons/Person';
+   ```
+
+2. Then update the `@backstage/core-components` import like this:
+
+   ```diff
+     import {
+       Sidebar,
+       sidebarConfig,
+       SidebarDivider,
+       SidebarGroup,
+       SidebarItem,
+       SidebarPage,
+       SidebarScrollWrapper,
+       SidebarSpace,
+       useSidebarOpenState,
+       Link,
+   +   GroupIcon,
+   +   SidebarSubmenu,
+   +   SidebarSubmenuItem,
+     } from '@backstage/core-components';
+   ```
+
+3. Finally replace `<SidebarItem icon={HomeIcon} to="catalog" text="Home" />` with this:
+
+   ```ts
+   <SidebarItem icon={HomeIcon} to="catalog" text="Home">
```

Comment:
So you can still click the "Home" item and go straight to the catalog right?

Reply:
Yes, that's correct, clicking on Home gets you to the same view as before